packages/web/components/complex/pools-table.tsx

```diff
@@ -36,6 +36,8 @@ import { api, RouterOutputs } from "~/utils/trpc";
 
 import { Tooltip } from "../tooltip";
 
+const FILTERABLE_IDS = ["2159"];
```

Comment:
Perhaps we can add a ternary operator that creates an array for testnet & mainnet.

Like:
`const FILTERABLE_IDS = IS_TESTNET ? [] : ["2159"];`